From 3b015a47be3e86b17dc8f0f05899ebbccbb97db1 Mon Sep 17 00:00:00 2001
From: radhak4x <radha.ravipatix.krishna@intel.com>
Date: Wed, 13 May 2020 00:34:26 +0530
Subject: [PATCH] UGW_SW-48467: Regulator shutdown failure case Recovery,
 update the counter Value as seen failure rarely in shutdown

---
 drivers/regulator/tps65273-regulator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/tps65273-regulator.c b/drivers/regulator/tps65273-regulator.c
index 3dee41770913..c9f5f098ad54 100755
--- a/drivers/regulator/tps65273-regulator.c
+++ b/drivers/regulator/tps65273-regulator.c
@@ -318,7 +318,7 @@ static void tps65273_shutdown(struct i2c_client *client)
 {
 	struct tps65273 *tps = i2c_get_clientdata(client);
 	int i, ret = 0;
-	int count = 5;
+	int count = 10;
 
 	dev_dbg(&client->dev, "function %s is called\n", __func__);
 
